src/biguint/convert.rs

```diff
@@ -65,9 +65,8 @@ fn from_inexact_bitwise_digits_le(v: &[u8], bits: u8) -> BigUint {
     debug_assert!(!v.is_empty() && bits <= 8 && big_digit::BITS % bits != 0);
     debug_assert!(v.iter().all(|&c| BigDigit::from(c) < (1 << bits)));
 
-    let big_digits = (v.len() as u64)
-        .saturating_mul(bits.into())
-        .div_ceil(&big_digit::BITS.into())
+    let big_digits = (v.len() as u64).saturating_mul(bits.into());
```

Comment:
This is kind of a style nit, but I don't like shadowing names like this when the first value doesn't make sense to have that name. This one could be something like `big_bits` or `total_bits`.